scripts/run_integration_tests_vfio.sh

```diff
@@ -18,10 +45,23 @@ fi
 
 cargo build --all --release $features --target $BUILD_TARGET
 strip target/$BUILD_TARGET/release/cloud-hypervisor
+strip target/$BUILD_TARGET/release/ch-remote
 
-export RUST_BACKTRACE=1
+# We always copy a fresh version of our binary for our L2 guest.
+cp target/$BUILD_TARGET/release/cloud-hypervisor $VFIO_DIR
+cp target/$BUILD_TARGET/release/ch-remote $VFIO_DIR
 
-time cargo test $features "vfio::$test_filter" -- --test-threads=1 ${test_binary_args[*]}
+# test_vfio rely on hugepages
+echo 6144 | sudo tee /proc/sys/vm/nr_hugepages
+sudo chmod a+rwX /dev/hugepages
+
+export RUST_BACKTRACE=1
+time cargo test $features "vfio::test_vfio" -- ${test_binary_args[*]}
 RES=$?
 
+if [ $RES -eq 0 ]; then
```

Comment:
I wonder is it required to run the nested `test_vfio` test first?

Reply:
Since the VFIO tests take longer to run than `test_vfio`, I thought it would be good to run `test_vfio` first, this way we would fail faster in case a regression is identified. But this is not required.